plugins/syslog/tee_plugin_method.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
This file should be made available from optee_client instead to avoid keeping possibly diverging copies.

Reply:
I droped this file.
Now we use `$(TEEC_EXPORT)/include` in syslog Makefile